runtime/src/main/java/org/corfudb/runtime/view/ObjectBuilder.java

```diff
@@ -90,6 +90,7 @@ public T open() {
 
         if (streamName != null) {
             streamID = CorfuRuntime.getStreamID(streamName);
+            log.info("CorfuObject: creating stream {} id {}", streamName, streamID);
```

Comment:
CorfuObject -> ObjectBuilder?

Reply:
Since this can appear in any vertical, having Corfu in the name I thought helps.

Reply:
Maybe we can put Corfu after this？ something like `"ObjectBuilder: creating Corfu stream ..."`

Reply:
done.